references/detection/train.py

```diff
@@ -108,20 +108,20 @@ def main(args):
 
     # lr_scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=args.lr_step_size, gamma=args.lr_gamma)
     lr_scheduler = torch.optim.lr_scheduler.MultiStepLR(optimizer, milestones=args.lr_steps, gamma=args.lr_gamma)
-
+    last_epoch = 0
     if args.resume:
         checkpoint = torch.load(args.resume, map_location='cpu')
         model_without_ddp.load_state_dict(checkpoint['model'])
         optimizer.load_state_dict(checkpoint['optimizer'])
         lr_scheduler.load_state_dict(checkpoint['lr_scheduler'])
-
+        last_epoch = lr_scheduler.last_epoch
```

Comment:
Just double-checking: should this be `lr_scheduler.last_epoch` or `lr_scheduler.last_epoch + 1`?

Reply:
I have the same idea with you at first, but I think this style is more simple and make less parameter to set. Most importantly, others won't make a mistake about which epoch to resume because the last epoch is kept by `lr_scheduler`(And we don't need to consider  if we set a wrong start epoch that will overwrite the checkpoint). For your convenience in merge, I will submit the second one for chosing. Thanks!